requirements/base.txt

```diff
@@ -1,6 +1,6 @@
 Jinja2
 jmespath
-msgpack>=0.5,!=0.5.5
+msgpack>=1.0.0
```

Comment:
We have not changed the megapack min version in along time. Should we add a small note the changelong?

Reply:
As @s0undt3ch pointed out 1.0.0 is already 3 years old. So I don't think we need the extra changelog.